src/core/ext/filters/client_channel/subchannel_stream_client.h

```diff
@@ -169,7 +166,7 @@ class SubchannelStreamClient
     grpc_metadata_batch send_initial_metadata_;
 
     // send_message
-    absl::optional<SliceBufferByteStream> send_message_;
+    absl::optional<SliceBuffer> send_message_;
```

Comment:
Similar comment as in surface/call.cc: Does this actually need to use `absl::optional<>`, or can we just create the `SliceBuffer` directly?  This is especially true in this case, since unlike the code in surface/call.cc, we know that we're going to send a message in this case.